projects/core/src/i18n/index.ts

```diff
@@ -1,5 +1,4 @@
 export * from './config/i18n-config';
-export * from './config/i18n-config-initializer';
```

Comment:
Why?

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files